src/server/replica.cc

```diff
@@ -117,6 +117,18 @@ error_code Replica::Start(ConnectionContext* cntx) {
   return {};
 }  // namespace dfly
 
+error_code Replica::EnableReplication(ConnectionContext* cntx) {
```

Comment:
this function always succeeds so error_code is not needed.
similarly, you use cntx to send ok but then you pass the string sink to mock the socket. seems redundant?